src/content/subgames.cpp

```diff
@@ -110,6 +116,11 @@ SubgameSpec findSubgame(const std::string &id)
 	std::set<std::string> mods_paths;
 	if (!user_game)
 		mods_paths.insert(share + DIR_DELIM + "mods");
+
+	Strfnd mod_search_paths(getModPathEnv());
+	while (!mod_search_paths.at_end())
+		mods_paths.insert(mod_search_paths.next(PATH_DELIM));
```

Comment:
I think this should be after `user + DIR_DELIM + "mods"` or is there a reason it needs to take precedence?

Reply:
I don't see a reason MINETEST_MOD_PATH should take precedence over mods installed from ContentDB in ~/.minetest.  The order you suggest seems reasonable.  I'll change the patch accordingly.